scripts/e2e-cluster-dump.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
+
+help() {
+  cat <<EOF
+This script generates logs for mayastor pods and cluster state.
+
+Usage: $0 [OPTIONS]
+
+Options:
+  --path <path>   Location to store log files
```

Comment:
For the sake of consistency across layers. Eases up orientation when reading if the same thing has the same name.
```suggestion
  --destdir <path>   Location to store log files
```

Reply:
Done